doc/api/package.json.md

```diff
@@ -0,0 +1,345 @@
+# `package.json`
+
+<!--introduced_in=REPLACEME-->
+<!-- type=misc -->
+
+## Introduction
+
+A `package.json` file defines a package scope. Its parent directory, and all
+subfolders below that folder down until the next folder containing another
+`package.json`, is considered a _package scope_.
+
+This document aims to describe the fields used by the Node.js
+runtime. Other tools (such as
+[npm](https://docs.npmjs.com/creating-a-package-json-file)) may use additional
+fields which are ignored by Node.js and not documented in this document.
+
+## Supported fields
+
+### `"name"`
+<!-- YAML
+added:
+  - v13.1.0
+  - v12.16.0
```

Comment:
This was added based on a user request (#33143). The introduction explicitly state that the documented fields are the one used by Node.js only. Do you suggest to just remove it or to make it clearer that it can be used alongside older versions of Node.js?